backend/protocol_rpc/endpoint_generator.py

```
@@ -61,6 +61,10 @@ def unfold(x: typing.Any):
         return repr(x)
 
 
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)